### DISCUSSION TITLE
Target Error

### DISCUSSION THREAD

**Discussion by sabarrag** [CS342/.github#10] [Technical Questions] [answered]
### In what area do you have a technical challenge?

Xcode, Simulator & Previews

### Description

Hi! I am not sure why I am getting this target error after I close the preview of the application. I thought I properly created my target and the build succeeds when I preview. Do you know what I could be doing wrong?

<img width="754" alt="image" src="https://user-images.githubusercontent.com/102919179/213897599-6f778b04-c397-40bc-bf5b-39b104fe5392.png">

### Reproduction

N/A

### Expected behavior

N/A

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment, marked as answer]
Hi @sabarrag,

Can you share your Swift Package Manifest file with us in a comment? GitHub allows you to share code snippets including syntax highlighting in comments and issues: [Creating Code Blocks - Syntax highlighting](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting).

Can you also check that your target is associated with a folder in the "Sources" folder of the Swift Package and that it contains a valid Swift file? 

Also try and see if the steps lined out here ["Package resolution errors must be fixed before building - @PSchmiedmayer's Comment"](https://github.com/orgs/CS342/discussions/6#discussioncomment-4742407) can maybe help you to resolve your issue.

**Reply by sabarrag** [reply]
The steps helped resolve my issue. Thank you so much!

**Reply by PSchmiedmayer** [reply]
You're welcome @sabarrag 👍
